### PR TITLE
Added container logs

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: flood.py [hunk 1]
```diff
@@ -29,6 +29,13 @@ def gen_json(hypervisors, guests):
 
 def rm_container(client, containers, reason="Success"):
     del_container = containers[0]
+    container_log = open("container.log", "a")
```
Comment (JacobCallahan) [inline flood.py]: Just curious why you don't use a context manager to work with the file.

Comment (sthirugn) [reply flood.py]: only reason is that I havent coded in python in a while ;) Updating now

**Comment by sthirugn** [comment]
@JacobCallahan thank you for the review, updated now

File: flood.py [hunk 2]
```diff
@@ -29,6 +29,13 @@ def gen_json(hypervisors, guests):
 
 def rm_container(client, containers, reason="Success"):
     del_container = containers[0]
+    with open("container.log", "a") as log:
+        log.write(
+            "***********************************{0}****************************\n".format(
+            del_container['name'])
+        )
+        log.write(client.logs(del_container['container']['Id']))
+        log.close()
```
Comment (JacobCallahan) [inline flood.py]: no need for the log.close() since you're operating within the context manager.

**Review by JacobCallahan — APPROVED** [review]
ACK

**Pull request merged**